src/components/icons/DownArrow.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+export const DownArrow = () => (
```

Comment:
i think we should clean these svgs - protip: https://jakearchibald.github.io/svgomg/ is awesome

Reply:
Nice tool!